lib/pbench/server/database/models/users.py

```diff
@@ -112,14 +113,15 @@ def delete(username):
             raise
 
     def is_admin(self):
-        # TODO: Add notion of Admin user
         """this method would always return false for now until we add a notion of Admin user/group.
         Once we know the admin credentials this method can check against those credentials to determine
         whether the user is privileged to do more.
 
         This can be extended to groups as well for example a user belonging to certain group has only those
         privileges that are assigned to the group.
         """
+        if self.entitlement in ["admin", "administrator"]:
```

Comment:
I don't think we need to support two names for the same role.  The CLI and/or UI can map between convenience names and the one internal role name.

Reply:
Funny, because I said the same thing ... and now I realize that I somehow didn't publish my comments. Sigh.